ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.10__amd64-nightly-4.10-upgrade-from-stable-4.10.yaml

```diff
@@ -130,6 +130,20 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test-cert-rotation
     workflow: cucushift-installer-rehearse-azure-aro
+- as: gcp-ipi-workload-identity-rt-fips-p2-f1
+  cron: 24 9 * * *
+  remote_api: true
```

Comment:
No need the `remote_api` now, as it change the defaults to true several days ago.